src/test/java/com/microsoft/sqlserver/UnitStatement/NamedParamMultiPart.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Microsoft JDBC Driver for SQL Server
+ * 
+ * Copyright(c) Microsoft Corporation All rights reserved.
+ * 
+ * This program is made available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.UnitStatement;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Types;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+@RunWith(JUnitPlatform.class)
+public class NamedParamMultiPart extends AbstractTest {
+    private static final String dataPut = "eminem ";
+    private static Connection Connection2 = null;
+    private static Driver Driver1 = null;
+    private static CallableStatement cs = null;
+
+    @BeforeAll
+    public static void beforeAll() throws SQLException {
+        Driver1 = DriverManager.getDriver(connectionString);
+        Connection2 = DriverManager.getConnection(connectionString);
+    }
+
+    @Test
+    @DisplayName("Named Param Multi Part Test")
+    public void datatypestest() throws Exception {
+        Connection Connection1 = DriverManager.getConnection(connectionString);
```

Comment:
Do you really need connection1 & statment1? Because in AbstractTest we open one connection and accessible for all child classes. If somehow this connection fails due to may be wrong db connectionstring etc then all testcases which extends AbstractTest get skipped due to failure of setup method. 

**For specific testing it is goo to create your own connection & close that one** but this case for connection1 we are not doing anything.